flixel/input/gamepad/FlxGamepadMapping.hx

```diff
@@ -88,16 +130,48 @@ class FlxGamepadMapping
 			{
 				case Logitech: LogitechID.RIGHT_ANALOG_STICK;
 				case OUYA: OUYAID.RIGHT_ANALOG_STICK;
-				case PS3: PS3ID.RIGHT_ANALOG_STICK;
 				case PS4: PS4ID.RIGHT_ANALOG_STICK;
 				case XBox360: XBox360ID.RIGHT_ANALOG_STICK;
 				case XInput: XInputID.RIGHT_ANALOG_STICK;
+				case MayflashWiiRemote: 
+					switch(attachment)
+					{
+						case WiiClassicController: MayflashWiiRemoteID.RIGHT_ANALOG_STICK;
+						default: null;
+					}
+				case WiiRemote: 
+					switch(attachment)
+					{
+						case WiiClassicController: WiiRemoteID.RIGHT_ANALOG_STICK;
+						default: null;
+					}
 				default: null;
 			}
 		}
 		return null;
 	}
 	
+	/**
+	 * Returns 1 or -1 depending on whether this axis needs to be flipped
+	 * @param	AxisID
```

Comment:
Might as well remove the `@param` and `@return` here, there's no description for them.